images/scripts/opensuse-tumbleweed.setup

```diff
@@ -78,6 +78,9 @@ rpm-config-SUSE \
 
 NETWORK_PACKAGES="\
 systemd-network \
+sssd \
+sssd-dbus \
```

Comment:
Is this just "for later", when you start testing cockpit? c-podman shouldn't care about sssd.

Reply:
When the tests were failing due to dbus configuration, it complained about not finding sssd service so I decided to throw them in, just in case.